gst/mqtt/mqttcommon.h

```diff
@@ -21,4 +21,30 @@
 #define GST_MQTT_ELEM_NAME_SINK "mqttsink"
 #define GST_MQTT_ELEM_NAME_SRC "mqttsrc"
 
+#define GST_MQTT_LEN_MSG_HDR    512
+
+/**
+ * @brief GST_BUFFER_MEM_MAX in gstreamer/gstbuffer.c is 16. To represent each
+ *        size of the memory block that the GstBuffer contains, GST_MQTT_MAX_NUM_MEMS
+ *        should be 16.
+ */
+#define GST_MQTT_MAX_NUM_MEMS   16
+
+/**
+ * @brief Defined a custom data type, GstMQTTMessageHdr
+ *
+ * GstMQTTMessageHdr contains the information needed to parse the message data
+ * at the subscriber side and is prepended to the original message data at the
+ * publisher side.
+ */
+typedef struct _GstMQTTMessageHdr {
+  union {
+    struct {
```

Comment:
MessageHdr may need version info in itself?

Reply:
@myungjoo Right. But at this point, we do not need it at all since it could be rapidly changed until the stable version.